docs/fastdds/security/access_control_plugin/access_control_plugin.rst

```diff
@@ -321,7 +324,7 @@ Topic expression
 
 This element is delimited by the ``<topic_expression>`` XML element tag.
 The value in this element identifies the set of Topic names to which the rule applies.
-The rule applies to any |DataReader| or |DataWriter| associated with a |Topic| whose name matches the value.
+The rule applies to any DataReader or DataWriter associated with a Topic whose name matches the value.
```

Comment:
Not here, but on the previous subsection (Topic Rule), right after the table, there are two links to Topic on the same sentence. Remove the second one.